control-plane-operator/hostedclusterconfigoperator/controllers/resources/konnectivity/params.go

```diff
@@ -56,6 +56,24 @@ func NewKonnectivityParams(hcp *hyperv1.HostedControlPlane, images map[string]st
 			SuccessThreshold:    1,
 		},
 	}
+
+	p.DeploymentConfig.ReadinessProbes = config.ReadinessProbes{
```

Comment:
Does this change cover both the control plane and data plane?  Is your intention to only change one or the other or both?

Reply:
the intention is to change both, on control plane and data plane